CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # node-google-dfp-wrapper
 
+### 1.3.0
+- Adds `getCriteriaValues` for querying criteria values without extracting id.
+- Adds `getCriteriaValueId` for querying criteria values and extracting id.
```

Comment:
I wonder if we could also just kinda overload `getCriteriaValues` with a parameter:

`getCriteriaValues(conditions)`   ->  gets values without ID
`getCriteriaValues(conditions, getId)` -> gets values with ID

The way that you have it is fine tho.